sdk/communication/Azure.Communication.Sms/src/SmsClient.cs

```diff
@@ -98,17 +100,19 @@ protected SmsClient()
         /// <param name="from">The sender's phone number that is owned by the authenticated account.</param>
         /// <param name="to">The recipient's phone number.</param>
         /// <param name="message">The contents of the message that will be sent to the recipient. The allowable content is defined by RFC 5724. If the message has more than 160 characters, the server will split it into multiple SMSs automatically.</param>
-        /// <param name="sendSmsOptions">Optional configuration for sending SMS messages.</param>
+        /// <param name="options">Optional configuration for sending SMS messages.</param>
         /// <param name="cancellationToken">The cancellation token for the task.</param>
         /// <exception cref="RequestFailedException">The server returned an error. See <see cref="Exception.Message"/> for details returned from the server.</exception>
         /// <exception cref="ArgumentNullException"><paramref name="from"/> is null.</exception>
         /// <exception cref="ArgumentNullException"><paramref name="to"/> is null.</exception>
         /// <exception cref="ArgumentNullException"><paramref name="message"/> is null.</exception>
-        public virtual async Task<Response<SendSmsResponse>> SendAsync(PhoneNumberIdentifier from, PhoneNumberIdentifier to, string message, SendSmsOptions sendSmsOptions = null, CancellationToken cancellationToken = default)
+        public virtual async Task<Response<SmsSendResult>> SendAsync(string from, string to, string message, SmsSendOptions options = default, CancellationToken cancellationToken = default)
         {
-            Argument.AssertNotNullOrEmpty(from.PhoneNumber, nameof(from));
-            Argument.AssertNotNullOrEmpty(to.PhoneNumber, nameof(to));
-            return await SendAsync(from, new[] { to }, message, sendSmsOptions, cancellationToken).ConfigureAwait(false);
+            Argument.AssertNotNullOrEmpty(from, nameof(from));
+            Argument.AssertNotNullOrEmpty(to, nameof(to));
+            Response<IEnumerable<SmsSendResult>> allSmsSendResults = await SendAsync(from, new[] { to }, message, options, cancellationToken).ConfigureAwait(false);
+            SmsSendResult sendSMSResult = allSmsSendResults.Value.FirstOrDefault();
```

Comment:
Let's use `First()`. The list should not be empty and hence the result should never be `null`.